pubsub/google/cloud/pubsub_v1/gapic/publisher_client.py

```diff
@@ -51,10 +51,10 @@
 
 
 # TODO: remove conditional import after Python 2 support is dropped
-if six.PY3:
-    from collections.abc import Mapping
-else:
+if six.PY2:
     from collections import Mapping
+else:
+    from collections.abc import Mapping
```

Comment:
Thanks, I'll update it. How do I re-generate?

Reply:
Done in 153d00385d004242e96741cdc81b5c0fbd08564b. Or is it enough that it's manually changed in the generated code this time, and next time it'll also be fine?

Reply:
Files can be re-generated with the [synth tool](https://github.com/googleapis/synthtool), its README contains instructions on how to use it.

Thanks again!

Reply:
In another branch (https://github.com/googleapis/google-cloud-python/compare/master...hugovk:fix-flake8-2020b), I reverted the change to the pubsub file and ran synthtool to regenerate the code. 

There were a lot of changes, but looks like it's because Black wasn't run due to this:

```
nox > Running session blacken
nox > Session blacken skipped: Python interpreter 3.6 not found.
```

I don't have Python 3.6 installed. How can I run it with, say, 3.7?

Reply:
When running the synth tool locally, I also saw a lot of changes, mostly black-related. On the plus side, the generated changes to `publisher_client` matched those that were added manually in this PR (no diff in that part).

@busunkim96 How about only committing the changes in the PubSub's synth file here (they look good), and re-generate the files in a separate PR to avoid the bloat here?

Reply:
@plamut That sounds good to me

Reply:
@hugovk Let's then revert the changes to the publisher client in the original commit, and keep those in the synth file, but without re-generating the code in this PR, and we should be good.

Reply:
Thanks, removed and rebased in 14014571a5398a5d5514752c2e5032d681550f07!